scripts/tests/test_validate_docstrings.py

```diff
@@ -255,29 +255,28 @@ def test_validate_all_ignore_errors(self, monkeypatch):
             ],
         )
 
-        exit_status_ignore_func = validate_docstrings.print_validate_all_results(
+        exit_status = validate_docstrings.print_validate_all_results(
             output_format="default",
             prefix=None,
-            errors=["ER01", "ER02"],
             ignore_deprecated=False,
-            ignore_errors={
-                "pandas.DataFrame.align": ["ER01"],
-                # ignoring an error that is not requested should be of no effect
-                "pandas.Index.all": ["ER03"]
-            }
+            ignore_errors={"*": {"ER03"}},
         )
+        # two functions * two not ignored errors
+        assert exit_status == 2 * 2
+
         exit_status = validate_docstrings.print_validate_all_results(
             output_format="default",
             prefix=None,
-            errors=["ER01", "ER02"],
             ignore_deprecated=False,
-            ignore_errors=None
+            ignore_errors={
+                "*": {"ER03"},
+                "pandas.DataFrame.align": {"ER01"},
+                # ignoring an error that is not requested should be of no effect
+                "pandas.Index.all": {"ER03"}
+            }
         )
-
-        # we have 2 error codes activated out of the 3 available in the validate results
-        # one run has a function to ignore, the other does not
-        assert exit_status == 2*2
-        assert exit_status_ignore_func == exit_status - 1
+        # two functions * two not global ignored errors - one function ignored error
+        assert exit_status == 2 * 2 - 1
```

Comment:
The diff of all this part seems a bit complex, but I just reordered the two calls since the test was a bit difficult to read before, as it was calling the two functions first, and then asserting the exit codes in the reverse order as they were being called. There is not change in logic other than replacing the `error` parameter with `ignore_errors` as in the rest.